sdk/core/core-tracing/src/index.ts

```diff
@@ -10,6 +10,7 @@ export { OpenCensusSpanWrapper } from "./tracers/opencensus/openCensusSpanWrappe
 export { OpenCensusTracerWrapper } from "./tracers/opencensus/openCensusTracerWrapper";
 export { TestTracer, SpanGraph, SpanGraphNode } from "./tracers/test/testTracer";
 export { TestSpan } from "./tracers/test/testSpan";
+export { createSpanFunction, CreateSpanFunctionArgs } from "./createSpan";
 
```

Comment:
should we consider reducing the public surface now and remove stuff that are no longer needed after this work?

Reply:
What are you thinking should be removed?

Reply:
I am not familiar with this package tbh, but perhaps we can remove things no longer in use by other package after your changes?

Reply:
(won't be in this PR since it's large enough for now)

Reply:
that is fine but please create an issue to track this to avoid any potential rot.

Reply:
Ah, okay. There's nothing at this time that I'm aware of. Let me know if you spot something.